src/App.tsx

```diff
@@ -1,39 +1,69 @@
 import React from 'react';
 import './App.scss';
+import { Timer } from './components/Timer';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+interface State {
+  clockName: string;
+  hide: boolean;
+}
+
+export class App extends React.Component<{}, State> {
+  state = {
+    clockName: 'Clock-0',
+    hide: true,
```

Comment:
`hide` property is confusing, you actually set it to `true` when showing clock, and set it to `false` when hiding, consider renaming it to something like `showClock`